ui/apps/platform/src/Containers/Clusters/InitBundles/SecureClusterUsingOperator.tsx

```diff
@@ -8,16 +12,93 @@ export type SecureClusterUsingOperatorProps = {
 function SecureClusterUsingOperator({
     headingLevel,
 }: SecureClusterUsingOperatorProps): ReactElement {
+    const { version } = useMetadata();
+    const subHeadingLevel = headingLevel === 'h2' ? 'h3' : 'h4';
+
     return (
         <Flex direction={{ default: 'column' }}>
             <Title headingLevel={headingLevel}>
                 Secure a cluster using Operator installation method
             </Title>
+            {version && (
+                <ExternalLink>
+                    <a
+                        href={getVersionedDocs(
+                            version,
+                            'installing/installing_ocp/install-secured-cluster-ocp.html'
+                        )}
+                        target="_blank"
+                        rel="noopener noreferrer"
+                    >
+                        Installing secured cluster services on Red Hat OpenShift
+                    </a>
+                </ExternalLink>
+            )}
+            <p>
+                You can install secured cluster services on your clusters by using the{' '}
+                <strong>SecuredCluster</strong> custom resource.
+            </p>
+            <Title headingLevel={subHeadingLevel}>
+                Procedure to do only once before you secure the first cluster
+            </Title>
             <List component="ol">
-                <ListItem>Do something.</ListItem>
-                <ListItem>Do something.</ListItem>
-                <ListItem>Do something.</ListItem>
+                <ListItem>
+                    <p>Use the OpenShift console to install the ACS Operator from Operator Hub.</p>
+                    <p>
+                        Create a new OCP project for ACS. A good name choice is{' '}
+                        <strong>rhacs-operator</strong>
+                    </p>
+                </ListItem>
+                <ListItem>
+                    <p>
+                        Download the YAML file for a cluster init bundle. You can use one bundle to
+                        secure multiple clusters.
+                    </p>
+                </ListItem>
+                <ListItem>
+                    <p>With the ACS project selected create the init bundle secrets in OCP.</p>
```

Comment:
I don't think this is correct. I think the init bundle secrets are created in the RHACS web portal, not in the OCP web console? Maybe I'm misunderstanding this text.

Reply:
Yeah, I do not see a corresponding step in the doc page. It is adapted from here:
https://github.com/RedHatInsights/acs-ui/blob/main/src/routes/GettingStartedPage/Wizard/InstallWithOperator.js#L91-L95

Reply:
I think it might be a mistake but I haven't run through the installation process in a long time. Someone in Eng could confirm.

Reply:
Simon, can you verify?

Reply:
We moved this from step 3 of **Prerequisites** to step 1 of **Procedure to repeat for each secured cluster**